ush/get_extrn_mdl_file_dir_info.sh

```diff
@@ -814,6 +824,10 @@ has not been specified for this external model:
 
       arcv_fns="${arcv_fns}gfs_pgrb2"
 
+    elif [ "${fv3gfs_file_fmt}" = "netcdf" ]; then
+
+      arcv_fns="${arcv_fns}gfs_pgrb2"
+
```

Comment:
Should this not be a netcdf file?

Reply:
This is a place hold. I added a comment on this.

Reply:
Done